packages/manager/modules/octavia-load-balancer/src/delete/delete.controller.js

```diff
@@ -0,0 +1,50 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(atInternet, $http, Alerter, $translate) {
+    this.isLoading = false;
+    this.atInternet = atInternet;
+    this.$http = $http;
+    this.Alerter = Alerter;
+    this.$translate = $translate;
+  }
+
+  delete() {
+    this.isLoading = true;
+    return this.$http
+      .delete(
+        `/cloud/project/${this.projectId}/region/${this.loadBalancerRegion}/loadbalancing/loadbalancer/${this.loadBalancerId}`,
+      )
```

Comment:
**nitpick:** This can be added in the service file so that it can be reused.

Reply:
Sorry I don't understand this one, I don't see any service in octavia app, should I create one?

Reply:
Yes, if the service does't exist. Create a new service and move the API fetching part to service so that it can be reused.